### ISSUE TITLE
[2.0.0] Phalcon\Tag::select() can not work with 'using' option in Volt

### ISSUE THREAD

**Issue by fukuda-s** [phalcon/cphalcon#10054]
Following code in Volt can not work using 2.0.0. (with 1.3.4 it works)

```
{{ select('productId', products, 'using': ['id', 'name'], 'useEmpty': true, 'emptyText': 'Please, choose one...', 'emptyValue': '@') }}
```
http://docs.phalconphp.com/en/latest/reference/tags.html#making-select-boxes

Thanks,

**Comment by andresgutierrez** [comment]
Are you getting an error or something?

**Comment by fukuda-s** [comment]
I can't find any massage on error log (Even the setting of error_reporting = E_ALL on php.ini).

Actually, the select form is shown on 2.0.0, however the second and third options (the value of "products" and "'using': ['id', 'name']" as following example code) don't work, then the select form options has empty names and values.

```
{{ select('productId', products, 'using': ['id', 'name'], 'useEmpty': true, 'emptyText': 'Please, choose one...', 'emptyValue': '@') }}
```

**Comment by FlexIDK** [comment]
If use in model 'colums'
\Model\Categories::find(array(
            'columns'   => array(
                'id', 'name', 'url_name',
            ), 'order' => 'name ASC',));
not work. select form options has empty names and values.

if use with out 'colums' \Model\Categories::find(array('order' => 'name ASC')); - all work correct

**Comment by pablomoreno61** [comment]
I have the same issue, but in my case, my products entity is a doctrine entity.

**Comment by GiTnEd** [comment]
Hello 
I am experiencing the same issue, posted my experience with it at this Thread -- #10284 
No errors being reported - Just blank select being formed.

**Comment by andresgutierrez** [comment]
I'm not able to reproduce this, could you please try again using latest Phalcon 2 (2.0.3) or submit a script that allow us to see the problem?

**Comment by fukuda-s** [comment]
Thank you for your comment.
I find another cause of this problem of code on controller.

If i used following code on controller, the problem does not occur.
```
$products = Products::find();
```

However, if I use "column" option on find() as follows, phalcon v2.0.3 returns blank select form.
```
$products = Products::find(array(
  'columns' => array(
    'id', 'name'
  ),
));
```

**Comment by andresgutierrez** [comment]
Fixed in Phalcon 2.0.4
